gradle/libs.versions.toml

```diff
@@ -1,6 +1,6 @@
 [versions]
 # Platform expectations
-paper = "1.20.4-R0.1-SNAPSHOT"
+paper = "1.21.4-R0.1-SNAPSHOT"
```

Comment:
Will that cause binary incompatibilities with older versions?

Reply:
no idea, could just go back to 1.20 there (was primarily required to see the errors / changes)